include/circt/Firtool/Firtool.h

```diff
@@ -260,6 +293,9 @@ struct FirtoolOptions {
   FirtoolOptions(llvm::cl::OptionCategory &category) : category(category) {}
 };
 
+LogicalResult populateLowerAnnotations(mlir::PassManager &pm,
```

Comment:
I think this should maybe be named slightly differently, since it does more than just lower annotations. Maybe something like `populatePreprocessingTransformations` or something like that.

Reply:
I have renamed it to `populatePreprocessTransforms` for a shorter name, what do you think?

Reply:
Makes sense to me.